src/lambda/local/configureLocalLambda.ts

```diff
@@ -126,16 +127,23 @@ function getTabSize(editor?: vscode.TextEditor): number {
     }
 }
 
-async function getSymbols(uri: vscode.Uri): Promise<vscode.DocumentSymbol[] | undefined> {
-    // Awaiting this command is required because without it, symbols for a newly created document might not yet
-    // be available, in which case vscode.executeDocumentSymbolProvider will return an empty list.
-    await vscode.commands.executeCommand<vscode.DocumentSymbol[]>('editor.action.wordHighlight.trigger', uri)
+async function loadSymbols(
+    uri: vscode.Uri,
+    maxAttempts = 3,
+    retryDelay = 500 // milliseconds
+): Promise<vscode.DocumentSymbol[] | undefined> {
+    if (maxAttempts <= 0) {
+        return undefined
+    }
 
-    return await vscode.commands.executeCommand<vscode.DocumentSymbol[]>('vscode.executeDocumentSymbolProvider', uri)
+    return await vscode.commands.executeCommand<vscode.DocumentSymbol[]>(
+        'vscode.executeDocumentSymbolProvider',
+        uri
+    ) || await sleep(retryDelay).then(async () => await loadSymbols(uri, maxAttempts - 1, retryDelay))
 }
 
 async function prepareConfig(editor: vscode.TextEditor, handler: string): Promise<boolean> {
-    const symbols: vscode.DocumentSymbol[] | undefined = await getSymbols(editor.document.uri)
+    const symbols: vscode.DocumentSymbol[] | undefined = await loadSymbols(editor.document.uri)
 
     // If the file is empty, or if it is non-empty but cannot be even partially parsed as JSON, build it from scratch.
     if (!symbols || symbols.length < 1) {
```

Comment:
As an extra line of defense, what do you think about an additional check here, if `symbols` is undefined, to see if the document contains non-whitespace? We could say to the user "hey, looks like there is a file, is it okay if we overwrite it?" <yes> <no (open file)> allowing them to back out. In this way:
* if the json was corrupt, they have a chance to inspect
* if there was json at all, the symbols should be loading for sure by now
* no files are harmed as best we can tell

Reply:
I agree with this suggestion. I will address it in a separate PR.